README.md

```diff
@@ -153,6 +153,7 @@ The app should then be available at http://localhost:5173/
 
 #### iOS and Android Simulators (with Hot Reload)
 
+Make sure to install Corvoda's required dependencies for [iOS](https://cordova.apache.org/docs/en/11.x/guide/platforms/ios/index.html)/[Android](https://cordova.apache.org/docs/en/11.x/guide/platforms/android/) before running the following commands.
```

Comment:
Hmm, could totally be incorrect, but I don't believe you should have to install these manually. Running `npm install` should be sufficient to install all the necessary dependencies, did you run into issues using that method?

Reply:
`npm install` installed all of the dependencies I needed except for the Android SDK, which I installed with Android Studio.
I think having these links could be convenient in the event that someone is struggling with the dependencies, so they could read about what they are missing, and I will mention Android Studio explicitly.
Do you think I should change the phrasing as well?

Reply:
Ahhh yeah okay, that makes sense! I think maybe changing the phrasing to something like "if you're having issues running the iOS/Android simulators, you may have to install these dependencies"

Reply:
I also encountered issues trying to run the iOS and Android servers locally, so I think these extra bits of documentation will be helpful! (That said, I haven't fully tried it yet)

P.s. in case the original wording is kept in some form, there's a typo: `Corvoda` -> `Cordova`